### PR TITLE
Update experiments to use the new `sparql-benchmark-runner`

### PR DESCRIPTION
**Pull request by surilindur**
This is a draft PR because it is still waiting for the `sparql-benchmark-runner` library to have a new version released, but the changes to the experiments could already be done.

### CODE REVIEW

File: packages/experiment-solidbench/lib/ExperimentSolidBench.ts [hunk 1]
```diff
@@ -183,6 +167,16 @@ export class ExperimentSolidBench implements Experiment {
     const network = networkHandler.network.id;
     const endpointProcessHandler = await this.hookSparqlEndpoint.start(context, { docker: { network }});
 
+    // Setup query loader and result serializer
+    const queryLoader = new QueryLoaderFile({
+      path: Path.join(context.experimentPaths.generated, 'out-queries'),
+      extensions: [ '.sparql' ],
+    });
+    const resultSerializer = new ResultSerializerCsv({
```
Comment (rubensworks) [inline packages/experiment-solidbench/lib/ExperimentSolidBench.ts:175]: I guess `ResultSerializerCsv` can be instantiated without args, as those are the default options?

**Comment by surilindur** [comment]
This should be ready for review now. I will make another PR for updating the SolidBench version used, when the PR for that one is merged eventually. :slightly_smiling_face:

**Review by rubensworks — APPROVED** [review]
👍 

Note to self: this requires a major release.

**Comment by rubensworks** [comment]
@surilindur I did not realize this before, but I see the CSV column names of the output of sparql-benchmark-runner have changed. For example, we used to have a `time` column, but this is now called `duration`.
Was this change deliberate? If not, can we revert it to the old column names, so all of our tooling that works on these CSV files is backwards-compatible?

**Comment by surilindur** [comment]
> Was this change deliberate? If not, can we revert it to the old column names, so all of our tooling that works on these CSV files is backwards-compatible?

It was deliberate at the moment, yes, but I totally forgot it. The reason I changed them was that `time` on its own is really non-specific, so either `executionTime` or `duration` made more sense. Maybe it is specific enough in the context of the results, though, and thereby works.

I have made a PR to revert it here: comunica/sparql-benchmark-runner.js#8

**Comment by surilindur** [comment]
This is now rebased on the master branch, and uses the latest benchmark runner library with reverted CSV column names.

**Comment by rubensworks** [comment]
Linter seems to fail 😢

**Comment by surilindur** [comment]
It worked locally, but I think there was something funny going on with the ESLint cache. After I deleted it, the linter failed. That is why I usually do not use the `--cache` flag, even for large projects, but maybe it is just a weird bug in an older version that has since been fixed. :thinking:

**Comment by surilindur** [comment]
The linter rules and Jest disagree on whether `undefined` is actually needed, I will need to fix that, still. :sweat_smile:

**Comment by rubensworks** [comment]
Thanks! Released as 5.0.0.

@surilindur Could you look into updating the jbr version over at https://github.com/comunica/comunica/tree/master/performance/benchmark-watdiv-file ?

**Pull request merged**